### PR TITLE
Project import generated by Copybara.

### PR DESCRIPTION
**Pull request by trav3711**
Project import generated by Copybara. Syncing tools/abel with github.

### CODE REVIEW

File: .github/workflows/tools.yml [hunk 1]
```diff
@@ -109,3 +109,15 @@ jobs:
...
         pylint --rcfile ./styles/pylintrc ./tools/rdf_generator
+      #---------- ABEL ----------#
+    - name: Run ABEL install dependencies
+      run: |
+        python setup.py install
+      working-directory: ./tools/abel
+    - name: ABEL tests
+      run: |
+        python -m unittest discover -p '*_test.py' --failfast -v
+      working-directory: ./tools/abel/tests
```
Comment (charbull) [inline .github/workflows/tools.yml:120]: break this out to create ABEL's workflow?

Comment (charbull) [reply .github/workflows/tools.yml:120]: or submit this workflow first to make sure the tests are running ok?

**Review by charbull — CHANGES_REQUESTED** [review]

File: .github/workflows/tools.yml (hunk 1 above)
Comment (trav3711) [reply .github/workflows/tools.yml:120]: Will do. Opened https://github.com/google/digitalbuildings/pull/776

Comment (trav3711) [reply .github/workflows/tools.yml]: Done.

File: tools/abel/model/entity_field.py [hunk 2]
```diff
@@ -32,7 +32,8 @@
 from model.units import Units
 
 
-# TODO(b/229631364) Extend to telemetry field and metadata field classes
```
Comment (charbull) [inline tools/abel/model/entity_field.py:35]: these should have been removed by copybara. interesting they are leaving outside. can you please remove them

File: tools/abel/model/export_helper.py [hunk 3]
```diff
@@ -208,7 +208,7 @@ def _GetConnections(self, entity: Entity) -> Dict[str, List[str]]:
     return {}
 
   def _SortLinks(self, entity: VirtualEntity) -> Dict[str, object]:
-    """Sorts an entity's links by guid and returns a Building Config compliant mapping.
+    """Sorts an entity's links by guid and returns mapping.
```
Comment (charbull) [inline tools/abel/model/export_helper.py:211]: a bc mapping?

File: tools/abel/model/model_builder.py [hunk 4]
```diff
@@ -123,9 +123,10 @@ def FromBuildingConfig(cls, site: Site,
       model_builder.AddReportingEntitiesFromEntityInstance(entity_instance)
     return model_builder
 
+  # pylint: disable=line-too-long
   # TODO(b/234630862) Refactor Build method for readability.
   def Build(self) -> None:
-    """Connects all entities to a site, fields to entities, and entities to entities based on attributes."""
+    """Connects ABEL graph with Guids as edges."""
```
Comment (charbull) [inline tools/abel/model/model_builder.py:129]: I prefer the prefer the previous docstring here

File: tools/abel/model/site.py [hunk 5]
```diff
@@ -22,6 +22,7 @@
 from model.entity import Entity
 
 
+# TODO(b/247621096): Combine site namespace and type name into one attribute.
```
Comment (charbull) [inline tools/abel/model/site.py:25]: remove?

**Review by charbull — CHANGES_REQUESTED** [review]

File: tools/abel/model/site.py (hunk 5 above)
Comment (trav3711) [reply tools/abel/model/site.py:25]: Done.

File: tools/abel/model/entity_field.py (hunk 2 above)
Comment (trav3711) [reply tools/abel/model/entity_field.py:35]: done.

File: tools/abel/model/export_helper.py (hunk 3 above)
Comment (trav3711) [reply tools/abel/model/export_helper.py:211]: Added back in but had to change format for line length.

File: tools/abel/model/model_builder.py (hunk 4 above)
Comment (trav3711) [reply tools/abel/model/model_builder.py:129]: Done. Same as above.

**Review by charbull — APPROVED** [review]

**Pull request merged**